PWGJE/EMCALJetTasks/macros/JetQA/runJetQA.sh

```diff
@@ -21,60 +21,106 @@
 # organized in the same file structure, i.e. a directory for each run, and $plotDir/TrainOutput should be the parent directory.
 #
 # Author: James Mulligan <james.mulligan@yale.edu>
-
-RUNLIST="246928 246846 246845 246844 246810 246809 246808 246807 246805 246804 246766 246765 246760 246759 246758 246757 246751 246750 246495 246493 246487 246434 246424 246271 246225 246222 246217 246115 246113 246089 246042 246037 246003 246001 245963 245954 245952 245949 245831 245829 245705 245702 245700 245683 246488 246087"
```

Comment:
Why do you delete this? It is (or was) the LHC15o list -- just put it below.

Reply:
Was an accident, is corrected in next commit